### PR TITLE
feat: Add GitVersion to sketchup CI run

### PR DESCRIPTION
**Pull request by AlanRynne**

### CODE REVIEW

File: .github/workflows/deploy.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# This workflow will install Python dependencies, run tests and lint with a single version of Python
+# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-python
+
+name: Build and deploy
+
+on:
+  push:
+    branches: ["dui3/alpha", "development", "main"] # Continuous delivery on every long-lived branch
```
Comment (oguzhankoral) [inline .github/workflows/deploy.yml:8]: Currently "development" and "main" dedicated for DUI2, we should not trigger DUI3 installer on them

**Review by oguzhankoral — CHANGES_REQUESTED** [review]

File: GitVersion.yml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+next-version: 3.0.0
+mode: ContinuousDeployment
+assembly-informational-format: "{Major}.{Minor}.{Patch}-{PreReleaseTag}"
+branches:
+  main:
+    regex: ^main$
```
Comment (oguzhankoral) [inline GitVersion.yml:6]: hmm, also don't know they are related to my previous comment

File: GitVersion.yml [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+    tag: rc
+  develop:
+    regex: ^development$
```
Comment (oguzhankoral) [inline GitVersion.yml:9]: hmm, also don't know they are related to my previous comment

File: GitVersion.yml (hunk 2 above)
Comment (AlanRynne) [reply GitVersion.yml:6]: Leaving this ones to have consistent config between our repos (it's the same as speckle-sharp-sdk and connectors).

It is required for gitversion to work properly

**Review by oguzhankoral — APPROVED** [review]

**Pull request merged**